week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -63,16 +58,16 @@ test(
 );
 test(
   "sortArray function works",
-  sortArray() === ["a", "c", "e", "f", "n", "z"]
+  JSON.stringify(sortArray()) === JSON.stringify(["a", "c", "e", "f", "n", "z"]) // you can not compare two arry only with "===" with JSON.stringify is possible
 );
-test("first5 function works", first5() === [1, 2, 3, 4, 5]);
+test("first5 function works", JSON.stringify(first5()) === JSON.stringify([1, 2, 3, 4, 5])); // you can not compare two arry only with "===" with JSON.stringify is possible
```

Comment:
You're right, we missed this. By the way, the usual way to do this is to compare them by checking every element in the array with a for loop.